config/git2consul.json

```diff
@@ -0,0 +1,25 @@
+{
+    "version": "1.0",
+    "repos" : [{
+        "name" : "config/store",
+        "url" : "https://github.com/ahmadiq/storeconfig.git",
```

Comment:
@ahmadiq can we put all configs in one repo? and then have different path? rite?